src/main/java/org/spongepowered/common/mixin/core/item/MixinItemBlock.java

```diff
@@ -35,7 +36,7 @@
 import java.util.Optional;
 
 @NonnullByDefault
-@Mixin(net.minecraft.item.ItemBlock.class)
+@Mixin(value = { net.minecraft.item.ItemBlock.class, net.minecraft.item.ItemAir.class })
```

Comment:
BlockTypes.AIR.getItem() no longer returns empty but an "air" item.
This changes the air itemType to return the air BlockType on getBlock() instead of empty Optional